### ISSUE TITLE
Fix for use with automerge 0.5

### ISSUE THREAD

**Issue by gterzian** [automerge/autosurgeon#31]
With automerge 0.5, when copying some structs from the example, I get errors like:

```
41  | #[derive(Debug, Clone, Reconcile, Hydrate, PartialEq)]
    |                                   ^^^^^^^ the trait `AsRef<automerge::exid::ExId>` is not implemented for `ObjId`
42  | struct Address {
43  |    line_one: String,
    |    ---------------- required by a bound introduced by this call
```

This goes away when going back to automerge 0.4

**Comment by bobby** [comment]
I'm having the same issue.  Our project needs some of the new features in automerge 0.5, including the new Patch API that replaces some of the older Observable API.

I started a PR to address this issue ([the branch](https://github.com/bobby/autosurgeon/tree/automerge-0.5.0)), but quickly became out of my depth trying to resolve Rust compiler issues related to automerge API changes.

I'm happy to help with this if needed.

**Comment by alexjg** [comment]
This is published as `autosurgeon` `0.8.0`

**Comment by bobby** [comment]
Kudos autosurgeon team, your turnaround time on issues is incredible!
